entries/normalizer.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0"?>
+<entry name="normalizer" type="method" return="String">
+	<title>normalizer</title>
+	<desc>Change/transform the value of the element before validating.</desc>
```

Comment:
I would use something like: "Prepares/transforms the elements value for validation."